README.md

```diff
@@ -1,6 +1,6 @@
 # Physics-Informed Neural Networks for Computational Mechanics (pinns_for_comp_mech)
```

Comment:
I messed up the reviewing history, this comment is a dummy comment :D

Reply:
no worries 😁